<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LianLianKanClass</class>
 <widget class="QMainWindow" name="LianLianKanClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>598</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LianLianKan</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/LianLianKan/Images/Background.jpg)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>530</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/start.jpg);
background-repeat: no-repeat;
margin-top: -2px;
margin-left: -2px;
}
QPushButton:pressed{
background: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resortButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/Resort.png);
background-repeat: no-repeat;
margin-top: 4px;
margin-left: -2px;
}
QPushButton:pressed{
background: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="navigateButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/Hint.png);
background-repeat: no-repeat;
margin-top: 4px;
margin-left: 5px;
}
QPushButton:pressed{
background: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="timeBar">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>549</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
border-width: 0px;
margin-left: -px;
margin-right: 0px;
margin-top: 0px;
margin-bottom: 0px;
background-color: transparent;
}

QProgressBar::chunk {
background: url(:/LianLianKan/Images/timebar.png);
margin-top: 1px;
margin-bottom: 1px;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="remainBlock">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>548</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="userInfo">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>74</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>昵称：游客
分数：0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>530</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/pause.png);
background-repeat: no-repeat;
margin-top: -4px;
margin-left: -4px;
}
QPushButton:pressed{
background: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="hardButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 
QPushButton:hover { 

}
QPushButton:pressed{
background: transparent;
}
&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;resortButton&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;640&lt;/x&gt;
     &lt;y&gt;160&lt;/y&gt;
     &lt;width&gt;31&lt;/width&gt;
     &lt;height&gt;31&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/Resort.png);
background-repeat: no-repeat;
margin-top: 4px;
margin-left: -2px;
}
QPushButton:pressed{
background: transparent;
}
&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string/&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="normalButton">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 

QPushButton:pressed{
background: transparent;
}
&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;resortButton&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;640&lt;/x&gt;
     &lt;y&gt;160&lt;/y&gt;
     &lt;width&gt;31&lt;/width&gt;
     &lt;height&gt;31&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/Resort.png);
background-repeat: no-repeat;
margin-top: 4px;
margin-left: -2px;
}
QPushButton:pressed{
background: transparent;
}
&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string/&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="easyButton">
    <property name="geometry">
     <rect>
      <x>723</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
}
QPushButton:pressed{
background: transparent;
}
&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QPushButton&quot; name=&quot;resortButton&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;640&lt;/x&gt;
     &lt;y&gt;160&lt;/y&gt;
     &lt;width&gt;31&lt;/width&gt;
     &lt;height&gt;31&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;styleSheet&quot;&gt;
    &lt;string notr=&quot;true&quot;&gt;QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/Resort.png);
background-repeat: no-repeat;
margin-top: 4px;
margin-left: -2px;
}
QPushButton:pressed{
background: transparent;
}
&lt;/string&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string/&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="versusButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background: transparent; border: none;} 
QPushButton:hover { 
background-image: url(:/LianLianKan/Images/versus.png);
background-repeat: no-repeat;
margin-top: -4px;
margin-left: -4px;
}
QPushButton:pressed{
background: transparent;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="opponentState">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/LianLianKan/Images/opponent.png);
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(44, 88, 160);
color: white;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LianLianKan.qrc"/>
 </resources>
 <connections/>
</ui>
